assets/admin/students/student-bulk-action-button/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { render, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import StudentModal from '../student-modal';
+
+/**
+ * Student bulk action button.
+ */
+export const StudentsBulkActionButton = () => {
+	const [ action, setAction ] = useState( 'add' );
+	const [ isModalOpen, setIsModalOpen ] = useState( false );
+	const closeModal = () => setIsModalOpen( false );
+	const setActionValue = ( selectedValue ) => {
+		switch ( selectedValue ) {
+			case 'enrol_restore_enrolment':
+				setAction( 'add' );
+				break;
+			case 'remove_enrolment':
+				setAction( 'remove' );
+				break;
+			case 'remove_progress':
+				setAction( 'reset-progress' );
+				break;
+			default:
+		}
+	};
+
+	const openModal = () => {
+		const hiddenSenseiBulkAction = document.getElementById(
+			'bulk-action-selector-top'
+		);
+		if ( hiddenSenseiBulkAction ) {
+			setActionValue( hiddenSenseiBulkAction.value );
+		}
+		setIsModalOpen( true );
+	};
+	return (
+		<>
+			<Button
+				className="button button-primary"
+				id="sensei-bulk-learner-actions-modal-toggle"
+				onClick={ openModal }
+				style={ { height: 30 } }
+			>
+				{ __( 'Select Courses', 'sensei-lms' ) }
+			</Button>
+			{ isModalOpen && (
+				<StudentModal action={ action } onClose={ closeModal } />
```

Comment:
@LavaGolem I think we should open the modal using the selected students.
On PR #5017 I [added](https://github.com/Automattic/sensei/blob/79ad9f0d78c5da9d3b412243b74e7dd6c8ef6945/assets/admin/students/student-modal/index.js#L74) the `students` attribute that is waiting for to receive the selected students.

Reply:
I've added students, this way looks simplest in terms of avoiding moving logic from the helper class for bulk actions.
[here](https://github.com/Automattic/sensei/pull/5004/commits/526441b5576c10a381cc5c42e24ed9f2810534d9)

Reply:
@gabrielcaires  ignore the message I've founf the issue, fixing it

Reply:
@gabrielcaires  I had to force the changes since merge broke some things: 
1. I've added students:
[here](https://github.com/Automattic/sensei/pull/5004/commits/4fd1e8e2a5f6a01eb7494ca831e93f208d82af34)
2. I've fixed the merge issues: 
[here](https://github.com/Automattic/sensei/pull/5004/commits/027d9fddb41393cc3528112e8d8c80cbedd65d71)